packages/webstudio-sdk/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@webstudio-is/sdk",
+  "version": "0.5.0",
+  "description": "Webstudio JavaScript / TypeScript API",
+  "author": "Webstudio <github@webstudio.is>",
+  "repository": "git@github.com:webstudio-is/webstudio-sdk.git",
+  "homepage": "https://webstudio.is",
+  "scripts": {
+    "build": "yarn build:mdn-data && yarn build:args && yarn build:prisma && yarn build:lib",
+    "build:mdn-data": "node --experimental-loader esbuild-node-loader ./bin/mdn-data.ts ./src/css",
+    "build:args": "node --experimental-loader esbuild-node-loader ./bin/generate-arg-types.ts './src/components/*.tsx !./src/**/*.stories.tsx'",
```

Comment:
lets check if we actually need those flags

Reply:
If I change the shebang from `#!/usr/bin/env` to `#!/usr/bin/env -S` it works.